openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -177,3 +177,12 @@ <h5 class="message__title">{% trans "We could not submit your response" %}</h5>
     {% endblock %}
 </li>
 {% endspaceless %}
+
+<script>
+$(document).ready(function() {
+    var savedLanguage = "{{saved_response.language}}";
```

Comment:
where is this saved_response.language coming from? Is it the draft/in progress code context data ?

Reply:
This is saved response on every Run Code. This is passed to the template when deciding what view should be loaded.